apps/website/src/components/EnhancedFAQ.tsx

```
@@ -0,0 +1,219 @@
+                                                        ? { opacity: 0 }
+                                                        : {
+                                                              height: 0,
+                                                              opacity: 0,
+                                                          }
+                                                }
+                                                transition={{ duration: 0.3 }}
+                                                className="overflow-hidden"
+                                            >
+                                                <div className="border-t border-gray-200 p-4 text-gray-600">
+                                                    <p
+                                                        dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)